bakerydemo/blog/models.py

```diff
@@ -188,7 +188,7 @@ def tag_archive(self, request, tag=None):
             return redirect(self.url)
 
         posts = self.get_posts(tag=tag)
-        context = {"tag": tag, "posts": posts}
+        context = {"page": self, "tag": tag, "posts": posts}
```

Comment:
Because of this not being set, the main bakerydemo doesn’t have breadcrumbs on blog tag indexes and doesn’t have the blog index as "active" in menus, which is likely unintended.